src/Operations/CompareMeshes.cc

```diff
@@ -167,16 +302,24 @@ bool CompareMeshes(Drover &DICOM_data,
                 P_B.x*P_A.y*P_C.z + P_A.x*P_B.y*P_C.z)/(6.0);
     }
 
-    
+    bool v_manifold_1 = IsVertexManifold(mesh1);
+    bool v_manifold_2 = IsVertexManifold(mesh2);
+    bool e_manifold_1 = IsEdgeManifold(mesh1);
+    bool e_manifold_2 = IsEdgeManifold(mesh2);
```

Comment:
Isn't mesh1 just the original mesh which should presumably be manifold? Doesn't hurt to check anyways I guess.

Reply:
apparently not when I checked, might be because of overlapping objects